mmv1/products/firestore/Index.yaml

```diff
@@ -96,6 +96,16 @@ properties:
     values:
       - :COLLECTION
       - :COLLECTION_GROUP
+      - :COLLECTION_RECURSIVE
+  - !ruby/object:Api::Type::Enum
+    name: apiScope
+    description: |
+      The API scope at which a query is run.
+    default_value: :ANY_API
+    values:
+      - :ANY_API
+      - :DATASTORE_MODE_API
+    ignore_read: true
```

Comment:
Question: why do we need to have `ignore_read` for this field? Does API not return values for this field?

Reply:
This is really for compatibility (though unclear to me what's the best way to do this in Terraform).

The goal is making ANY_API as default value (and hide if it's default). Only show API_SCOPE if it's DATASTORE_MODE_API. Please let me know if there is a better way to achieve this.

Reply:
I've seen you add a terraform side default `default_value: :ANY_API` already, which means that if users do not specify this fields, terraform will send `ANY_API`. Is `ANY_API` also the service side default for the field?
Plus, will API return the value back if users specify either `ANY_API` or `DATASTORE_MODE_API` for the field. 
If so, we'd remove `ignore_read` as this will prevent from reading back from the API.

Reply:
SG. Thanks for the suggestions!